core/src/Streamly/Internal/Data/Stream/StreamD.hs

```diff
@@ -1,42 +0,0 @@
--- |
```

Comment:
We can deprecate this module and export Streamly.Internal.Data.Stream from it. So that if anyone has imported this, it continues working.